mini-project-1/products.json

```diff
@@ -2,20 +2,203 @@
   {
     "productId": 1,
     "productName": "Starbucks Espresso",
-    "price": 99,
+    "price": [59, 99, 129],
     "description": "Dark roasted beans for your delightful morning...",
-    "category": ["Dark roast", "SML"], //SML stands for available sizes
+    "category": ["Dark roast", "SML", "Starbucks"],
```

Comment:
I don't think we have a reason to have an array here. We know for example that this is a Starbucks espresso. 
So this value  should be `"category": "Espresso"`.
Then, we can use the filter method provided by JS to display only the products that belong under this category.
Same with all the other objects: `"category": "Medium Roast"`